aspnetcore/tutorials/signalr-typescript-webpack.md

```diff
@@ -24,7 +24,22 @@ In this tutorial, you learn how to:
 
 [View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/tutorials/signalr-typescript-webpack/sample) ([how to download](xref:index#how-to-download-a-sample))
 
-[!INCLUDE [Prerequisites](~/includes/net-core-prereqs-vs-vsc-2.2.md)]
+## Prerequisites
+
+# [Visual Studio](#tab/visual-studio)
+
+* [Visual Studio 2017 version 15.9 or later](https://www.visualstudio.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=button+cta&utm_content=download+vs2017) with the **ASP.NET and web development** workload
+* [.NET Core SDK 2.2 or later](https://www.microsoft.com/net/download/all)
```

Comment:
Do we know the tutorial will work with 3.0?  If not maybe we should drop "or later".

Reply:
It worked for me in the latest preview, though I was using VS 2019. When 3.0 is officially released, we'll make 2019 the minimum.